pyk/src/pyk/kcfg/show.py

```diff
@@ -74,12 +71,12 @@ def node_attrs(self, kcfg: KCFG, node: KCFG.Node) -> list[str]:
 
 
 class KCFGShow:
-    kprint: KPrint
+    pretty_printer: PrettyPrinter
     node_printer: NodePrinter
```

Comment:
What's the difference between `pretty_printer` and `node_printer`? One is for displaying edges, the other is for nodes?

Reply:
Ideally, we wouldn't have `NodePrinter` at all. This change is what I could do to lower from `KPrint` to `KDefinition` in this class now. Later refactoring will remove the `print_node` from `NodePrinter`, and then it will be more like `NodeAttrs` instead, which ideally would be inlined directly into `KCFG` class (thus removing the `NodePrinter` heirarchy). For now, I know I can remove `NodePrinter.print_node`, but not sure about inlining into `KCFG`.